components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/utils/ServiceCatalogUtils.java

```diff
@@ -372,6 +388,11 @@ private static boolean processMetadata(File tempDir, File metadataFolder, File m
                                            Map<String, String> md5MapOfAllService) throws IOException
             , ResolverException, NoSuchAlgorithmException {
         String metaFileName = metadataYamlFolder.getName();
+        // check metadata is relevant to proxy-service
+        if (metaFileName.contains(PROXY_SERVICE_SUFFIX) && !(new File(metadataFolder,
```

Comment:
would endsWith be better than contains?

Reply:
We cannot use 'endsWith' here because metaFileName value is `<PROXY_NAME>_proxy_metadata_<VERSION>`